src/ImageSharp/Processing/Quantization/OctreeQuantizer.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing.Dithering;
+using SixLabors.ImageSharp.Processing.Dithering.ErrorDiffusion;
+using SixLabors.ImageSharp.Processing.Quantization.FrameQuantizers;
+
+namespace SixLabors.ImageSharp.Processing.Quantization
+{
+    /// <summary>
+    /// Allows the quantization of images pixels using Octrees.
+    /// <see href="http://msdn.microsoft.com/en-us/library/aa479306.aspx"/>
+    /// </summary>
+    public class OctreeQuantizer : IQuantizer
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OctreeQuantizer"/> class.
```

Comment:
Shouldn the docs mention that dithering is enabled with this ctr? (I'm following the twitter conversaton :) )

Reply:
Yeah they should....😋